documentation/modules/ROOT/partials/modules/all-connectors/con-connector-blocking-snapshot.adoc

```diff
@@ -16,10 +16,11 @@ After the snapshot completes, the streaming is resumed.
 
 You can set the following properties in the `data` component of a signal:
 
- * data-collections: to specify which {data-collection}s must be snapshot
+ * data-collections: to specify which {data-collection}s must be snapshot.
+   As for `table.include.list`, it is a comma-separated list of regular expressions that match fully-qualified {data-collection} identifiers.
```

Comment:
Don't know if in that case it is clear that `data-collections` act as the `table.include.list`. Am I wrong?

Reply:
That's a good point, what about this:
```suggestion
* data-collections: You specify which {data-collection}s that should be snapshot.
 ** This should be a comma-separated list of regular expressions that match fully-qualified {data-collection} names.
    It will be used in similar to `table.include.list`, allowing the blocking snapshot to snapshot one or multiple tables.
```

Reply:
```suggestion
* data-collections: Specifies the {data-collection}s that you want the snapshot to include.
 ** This property accepts a comma-separated list of regular expressions that match fully-qualified {data-collection} names.
The behavior of the property is similar to the behavior of the `table.include.list` property, which specifies the tables to capture in a blocking snapshot.
```